modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2413,6 +2445,19 @@ public void testGetObjectEntries() throws Exception {
 			).put(
 				"pt_BR", 54L
 			).build()
+		).put(
+			"localizedPicklistObjectFieldName_i18n",
+			HashMapBuilder.put(
+				"en_US",
+				HashMapBuilder.put(
+					"key", _listTypeEntryKey2
+				).build()
+			).put(
+				"pt_BR",
+				HashMapBuilder.put(
+					"key", _listTypeEntryKey1
+				).build()
+			).build()
```

Comment:
Will this test fail if we change?
```
"localizedPicklistObjectFieldName_i18n",
	HashMapBuilder.put(
		"en_US",
		HashMapBuilder.put(
			"key", _listTypeEntryKey1
		).build()
	).put(
		"pt_BR",
		HashMapBuilder.put(
			"key", _listTypeEntryKey2
		).build()
```

If the test doesn't fail, I think Brian would prefer `_listTypeEntryKey1` with `en_US`

Reply:
Hey @marcelabc ,

Initially, I set it as you suggested (en_US -> _listTypeEntryKey1 / pt_BR -> _listTypeEntryKey2). However, in other scenarios, I needed to test it with different values, so I adjusted the order accordingly.

Let me know if you'd like me to change it.